### PR TITLE
PD-2531 Add right aligned contact counts to the drop down.

### PR DESCRIPTION
**Pull request by mklauber**
image

### CODE REVIEW

File: gulp/webpack.config.js [hunk 1]
```diff
@@ -39,12 +39,12 @@ module.exports = {
       },
     ],
     // ie8 catchall. Some imported react components need this.
-    postLoaders: [
-      {
-        test: /\.js$/,
-        loaders: ['es3ify'],
-      },
-    ],
+//    postLoaders: [
```
Comment (tdphillips) [inline gulp/webpack.config.js]: Accidental?

Comment (darrint) [inline gulp/webpack.config.js]: Shouldn't commit this.

File: gulp/src/nonuseroutreach/components/SearchDrop.jsx [hunk 2]
```diff
@@ -157,6 +157,9 @@ export default class SearchDrop extends Component {
               className={classnames(
                   {'active': i === activeIndex})}>
               {result.display}
+              <span className="partner-count">
+                {result.count}
```
Comment (darrint) [inline gulp/src/nonuseroutreach/components/SearchDrop.jsx]: Should be something like `({result.count} contacts)`. Refer to the design.

Comment (mklauber) [inline gulp/src/nonuseroutreach/components/SearchDrop.jsx]: Fixed.

**Pull request merged**